Description: Disable test_sql completely as contains time dependent tests
 Disable test_sql tests as they contain a time dependent component
 which caused a FTBFS starting in 2019-01-01.
 .
 Although this was addressed upstream in the concrete case it will
 not suffice for the complete lifetime of buster.
Origin: vendor
Bug-Debian: https://bugs.debian.org/918153
Forwarded: no
Author: Salvatore Bonaccorso <carnil@debian.org>
Last-Update: 2019-09-02

--- a/test/Makefile.am
+++ b/test/Makefile.am
@@ -387,7 +387,6 @@ TESTS = \
 	test_scripts.sh \
 	test_sessions.sh \
 	test_shlexer.sh \
-	test_sql.sh \
 	test_sql_coll_func.sh \
 	test_sql_json_func.sh \
 	test_sql_fs_func.sh \
@@ -400,6 +399,7 @@ TESTS = \
 DISABLED_TESTS = \
 	test_top_status \
     test_logfile.sh \
+    test_sql.sh \
     test_view_colors.sh \
     test_vt52_curses.sh
 
